code/modules/clothing/head/helmet.dm

```diff
@@ -58,16 +58,26 @@
 	body_parts_covered = HEAD|FACE|EARS
 	siemens_coefficient = 1
 
-/obj/item/clothing/head/helmet/tactical
-	name = "tactical helmet"
+/obj/item/clothing/head/helmet/NT
```

Comment:
Seems to be an oversight here, there was a request to genericize this which was resolved, but it still remains an improper named item.